.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - image: circleci/python:2.7
     working_directory: ~/st2-packages
     environment:
-      DISTROS: "xenial bionic el6 el7 el8"
```

Comment:
As we just removed `el6` distro, please also adjust the CircleCI job parallelism above from `5` -> `4`